scripts/network-json-validator

```diff
@@ -417,18 +418,9 @@ def validate_networks databag
     end
   end
 
-  if not @networks['public'].subnet_addr_full.include?(@networks['nova_floating'].subnet_addr_full)
-    raise "'nova_floating' network must be a subnetwork of 'public' network"
-  end
-
-  if @networks['public'].use_vlan != @networks['nova_floating'].use_vlan
-    raise "The 'Use VLAN' setting (JSON attribute: 'use_vlan') of the 'nova_floating' and 'public' networks must be the same."
-  end
-
-  if @networks['public'].use_vlan
-    if @networks['public'].vlan != @networks['nova_floating'].vlan
-      raise "'nova_floating' network must have the same VLAN configuration as 'public' network"
-    end
+  if not @networks['public'].subnet_addr_full.include?(@networks['nova_floating'].subnet_addr_full) &&
```

Comment:
Though ```unless``` does the wrong thing here. Just as ```not```.  We want the ```not``` to just apply to the result of the ```include?``` call. ```!``` will just do that, as it has a higher precedence than ```&&```